Python/pywarpx/_libwarpx.py

```diff
@@ -1713,6 +1740,11 @@ def get_mesh_charge_density_fp_lovects(level, include_ghosts=True):
     '''
     return _get_mesh_array_lovects(level, None, include_ghosts, libwarpx.warpx_getChargeDensityFPLoVects)
 
+def get_particle_boundary_buffer_n_species():
+    return libwarpx.warpx_getParticleBoundaryBufferNSpecies()
+
+def get_num_particles_impacted_boundary(species_name, boundary):
```

Comment:
Also add a docstring here please.

Reply:
And then I guess you can use the same approach as above to make the boundary argument a string.